tests/components/test_streams_app.py

```diff
@@ -429,3 +429,62 @@ async def test_should_clean_streams_app_and_deploy_clean_up_job_and_delete_clean
                 "test-namespace", self.STREAMS_APP_CLEAN_NAME, dry_run
             ),
         ]
+
+    @pytest.mark.asyncio
+    async def test_get_topics(
```

Comment:
perhaps you can combine the two tests as the name suggests?

Reply:
what do you mean actually?

Reply:
would it make sense to put the two tests `test_get_topics` (currently it's more like `test_get_input_topics`) and `test_get_output_topic` together?